cli/src/funTest/kotlin/ExamplesTest.kt

```diff
@@ -117,6 +118,14 @@ class ExamplesTest : StringSpec() {
             }
         }
 
+        "how-to-fix-text-provider.kts can be run" {
+            val script = takeExampleFile("how-to-fix-text-provider.kts").readText()
+
+            HowToFixTextProvider.fromKotlinScript(script)
+
+            // TODO: It should also be verified that the script works as expected.
```

Comment:
Yes, please do, it's easy to do, isn't it?

Reply:
Done.